com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraUI.PresetInspector.cs

```diff
@@ -0,0 +1,36 @@
+using System.Linq;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+namespace UnityEditor.Rendering.Universal
+{
+    using CED = CoreEditorDrawer<UniversalRenderPipelineSerializedCamera>;
+
+    static class UniversalRenderPipelineCameraUIPreset
```

Comment:
A lot of the access modifiers are changed from private to internal or public. Could we avoid this by integrating the Preset inspector into `UniversalRenderPipelineCameraUI`? This could be a nested class or not. 
In `UniversalRenderPipelineCameraEditor.cs` it could look something along the lines of:
`UniversalRenderPipelineCameraUI.PresetInspector.Draw(m_SerializedCamera, this);`

As a result of this all drawers could remain private. Same for the Styles in the Skin

Reply:
we could. i dont really understand the benefit but if thats what we want for URP codebase I dont mind updating the code.
@alex-vazquez any opinion on this? you have seen more URP code than I

Reply:
I think @svens-unity is having a good point here. If we are avoiding making public API is always a good path to follow.

Reply:
okay then! thanks for the feedback, i ll work on this asap